addons/func_godot/src/trenchbroom/trenchbroom_game_config.gd

```diff
@@ -43,14 +50,21 @@ extends Resource
 @export_category("Editor Hint Tags")
 
 ## TrenchBroomTag resources that apply to brush entities.
-@export var brush_tags : Array[Resource] = []
+@export var brush_tags : Array[Resource] = [
+	preload("res://addons/func_godot/game_config/trenchbroom/tb_brush_tag_func.tres"),
+	preload("res://addons/func_godot/game_config/trenchbroom/tb_brush_tag_trigger.tres")
+]
 
 ## TrenchBroomTag resources that apply to brush faces.
 @export var brushface_tags : Array[Resource] = [
 	preload("res://addons/func_godot/game_config/trenchbroom/tb_face_tag_clip.tres"),
 	preload("res://addons/func_godot/game_config/trenchbroom/tb_face_tag_skip.tres")
 ]
 
+@export_category("Compatibility")
+## Keeps track of current Trenchbroom version. It's located above the New Map button when you open Trenchbroom.
```

Comment:
That's not an accurate description anymore.
`## Game configuration format compatible with the version of TrenchBroom being used.` would be more accurate. 
Avoid tutorialization in the property descriptions.